### PR TITLE
Xcode 8.3 compatibility tweaks

### PR DESCRIPTION
**Pull request by steam**
Xcode 8.3 introduced a new compile flag `-enable-bridging-pch` which has sped up my local compile times significantly. 🎉 

Additionally, several changes were required to get both the app and specs compiling correctly. After discussing inconsistent snapshot specs in Xcode 8.3, Lucian and I decided to accept the new ones and make adjustments as necessary in future pull requests.

### CODE REVIEW

**Comment by rynbyjn** [comment]
Should we up the image value in the `.travis.yml` file, or do y'all typically wait until Travis has that image _actually_ available

**Comment by steam** [comment]
Yeah, good question @rynbyjn. Their [documentation](https://docs.travis-ci.com/user/osx-ci-environment/) does say:

>At this time we are unable to provide pre-release versions of Xcode due to the NDA imposed on them. We do test them internally, and our goal is to make new versions available the same day they come out.

I'll add it since this won't work on pre 8.3 anyway.

**Comment by steam** [comment]
>Executed 2578 tests, with 0 failures (0 unexpected) in 259.380 (260.832) seconds

File: Specs/Controllers/Drawer/DrawerViewDataSourceSpec.swift [hunk 1]
```diff
@@ -14,7 +14,7 @@ class DrawerViewDataSourceSpec: QuickSpec {
 
             describe("tableView(_:numberOfrowsInSection:)") {
 
-                it("returns 7") {
+                it("returns the correct number") {
                     let dataSource = DrawerViewDataSource()
                     expect(dataSource.tableView(UITableView(frame: .zero), numberOfRowsInSection: 0)) == 6
```
Comment (colinta) [inline Specs/Controllers/Drawer/DrawerViewDataSourceSpec.swift]: `"returns 7"` -> `)) == 6`.  Yup. 😃

Comment (steam) [inline Specs/Controllers/Drawer/DrawerViewDataSourceSpec.swift]: I almost changed it to "6" but decided to future proof it a bit. 😏

**Comment by colinta** [comment]
:+1:

**Pull request merged**